pkg/reporeader/reporeader.go

```diff
@@ -25,6 +26,15 @@ type TestRepoReader struct {
 	Files map[string][]byte
 }
 
+// GetRepoName implements RepoReader.
+func (TestRepoReader) GetRepoName() (string,error) {
```

Comment:
💡initial thoughts-  why the “test” will reside at the main package level?

Reply:
this seems to be an issue of semantics, since the intent of this reader is more of a "FakeRepoReader" than a "testRepoReader" but we could move it into the test package since it is only used for testing